src/web_peer_connection.cpp

```diff
@@ -578,8 +578,8 @@ void web_peer_connection::handle_error(int const bytes_left)
 	// associated with the file we just requested. Only
 	// when it doesn't have any of the file do the following
 	// pad files will make it complicated
-	int retry_time = atoi(m_parser.header("retry-after").c_str());
-	if (retry_time <= 0) retry_time = m_settings.get_int(settings_pack::urlseed_wait_retry);
+	int const retry_time = m_parser.header_uint("retry-after", 
```

Comment:
I prefer the comma on the new line, to make it clearer that this line is a continuation of the statement from the previous line.